packages/js/docs/swc-examples.md

```diff
@@ -0,0 +1,49 @@
+## Examples
+
+{% tabs %}
+{% tab label="Inline libraries" %}
```

Comment:
I see you are using tabs here but only one is used, is there a second one coming?

Reply:
copy-pasta. Added another one.